docs/formats.md

```diff
@@ -15,10 +15,45 @@ Need help displaying raster tiles in a map? We have several [examples](build-a-m
 
 **Terrarium** format _PNG_ tiles contain raw elevation data in meters, in Web Mercator projection (EPSG:3857). All values are positive with a 32,768 offset, split into the red, green, and blue channels, with 16 bits of integer and 8 bits of fraction.
 
+In other words, the red channel encodes the "256s" place, the green channel the "1s" place, and the blue channel the fractional component, which is 0 - 0.99609375 (255/256) in increments of 0.00390625 (1 / 256).
+
 To decode:
 
   `(red * 256 + green + blue / 256) - 32768`
 
+To encode, asuming a starting value of `v`:
+
+```
+v += 32768
+r = floor(v/256)
+g = floor(v % 256)
+b = floor(v % 256 % 1 * 256)
```

Comment:
I'm fine with `b = floor((v - floor(v)) * 256)`, making that change and then merging.